src/vendor/chartJs/utils.js

```diff
@@ -63,14 +68,28 @@ const generateOptions = (rawOptions = {}, data) => {
       },
     ];
   })();
-  const yMax = niceCeiling(
-    mostlyMax(
-      selectFrom(data.datasets)
-        .select('data')
-        .flatten()
-        .select('y')
-    )
-  );
+  const yMax = (() => {
+    const requestedMax = Data.get(options, 'axis.y.max');
+
+    if (isNumeric(requestedMax)) {
+      return requestedMax;
+    }
+
+    const niceMax = niceCeiling(
+      mostlyMax(
+        selectFrom(data.datasets)
+          .select('data')
+          .flatten()
+          .select('y')
+      )
+    );
+
+    if (isData(requestedMax)) {
+      return min([requestedMax.max, max([niceMax, requestedMax.min])]);
+    }
+
+    return niceMax;
+  })();
```

Comment:
Allow us to set a desired `max` and `min` on a chart (convert form standard form to ChartJS form)